naga/src/back/hlsl/conv.rs

```diff
@@ -122,7 +122,7 @@ impl crate::StorageFormat {
             Self::R16Float | Self::R32Float => "float",
             Self::R8Unorm | Self::R16Unorm => "unorm float",
             Self::R8Snorm | Self::R16Snorm => "snorm float",
-            Self::R8Uint | Self::R16Uint | Self::R32Uint => "uint",
+            Self::R8Uint | Self::R16Uint | Self::R32Uint | Self::R64Uint => "uint",
```

Comment:
Shouldn't this be `uint64_t`?

According to this it seems so https://microsoft.github.io/DirectX-Specs/d3d/HLSL_SM_6_6_Int64_and_Float_Atomics.html